addon/pods/components/frost-list-item/component.js

```diff
@@ -12,18 +12,30 @@ export default Ember.Component.extend({
   isSelected: Ember.computed.reads('model.isSelected'),
 
   onclick: Ember.on('click', function (event) {
-    if (!Ember.ViewUtils.isSimpleClick(event)) {
+    if (!Ember.ViewUtils.isSimpleClick(event) && !event.shiftKey && !event.metaKey && !event.ctrlKey) {
```

Comment:
I'm trying to keep the function running when shiftKey or CtrlKey is pressed even it's not a SimpleClick.